pkg/controllers/openshift-controller-manager.go

```diff
@@ -102,12 +103,14 @@ servingInfo:
 
 func (s *OCPControllerManager) Run(ctx context.Context, ready chan<- struct{}, stopped chan<- struct{}) error {
 	defer close(stopped)
+	errorChannel := make(chan error, 1)
 
 	// run readiness check
 	go func() {
 		healthcheckStatus := util.RetryTCPConnection("127.0.0.1", "8445")
 		if !healthcheckStatus {
-			klog.Fatalf(s.Name(), fmt.Errorf("healthcheck status"), "%s failed to start")
+			klog.Errorf(s.Name(), fmt.Errorf("healthcheck status"), "%s failed to start")
```

Comment:
This log format doesn't look right.  Errorf() takes a format string first, which here is passed as the last arg.

Reply:
oh right, I was just changing severity, but you're right.